src/WindowsTracing/include/WindowsTracing/TracerListener.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef WINDOWS_TRACING_TRACER_LISTENER_H_
+#define WINDOWS_TRACING_TRACER_LISTENER_H_
+
+#include "capture.pb.h"
+
+namespace orbit_windows_tracing {
+
+class TracerListener {
```

Comment:
Could you also add a class level comment here?

Reply:
Done.